build.sbt

```diff
@@ -1,9 +1,12 @@
 val previousVersion = "1.0.0"
 
+Global / excludeLintKeys += crossSbtVersions // used by a command but no other task
```

Comment:
Hum, this seems to be an SBT bug, no? I mean what we do here is the most basic usage of `crossSbtVersions`. Maybe file it upstream and reference here?

Reply:
Indeed. I submitted https://github.com/sbt/sbt/issues/6571